src/viur/core/modules/user.py

```diff
@@ -799,7 +799,7 @@ def start(self):
         return self._user_module.render.edit(
             self.OtpSkel(),
             params={
-                "name": i18n.translate(self.NAME),
+                "name": i18n.translate("viur.core.modules.user." + self.NAME),
```

Comment:
```suggestion
                "name": i18n.translate(f"viur.core.modules.user.{self.NAME}"),
```

and for others please use f-string too